packages/renderer/src/lib/pod/DeployPodToKube.svelte

```diff
@@ -192,10 +204,12 @@ async function deployToKube() {
       const createdRoute = await window.openshiftCreateRoute(currentNamespace, route);
       createdRoutes = [...createdRoutes, createdRoute];
     }
+    window.telemetryTrack('deployToKube', eventProperties);
 
     // update status
     updatePodInterval = setInterval(updatePod, 2000);
   } catch (error) {
+    window.telemetryTrack('deployToKube.error', { ...eventProperties, errorMessage: error.message });
```

Comment:
Why don't we have a single event ?

Reply:
I'm not sure to what you're referring when you say a single event
is it one 'name' or is it one `telemetryTrack` call

about the call one to send if there is an error and one if there is success.
About the name, it seems that errors are often logged with a `.error` suffix on the telemetry backend to quickly collect all errors

Reply:
I've changed to a single name.

Reply:
IMHO this will be better - easy to track overall use (# events), success rate (# errorMessage != null/# events), or see errors (errorMessage != null) via a single event. FWIW this is what I was trying to suggest for PR #1997 too.

Reply:
yes I've changed but I'm still not convinced. I think it really depends on the role.